### PR TITLE
add container flag to cli

### PR DESCRIPTION
**Pull request by dinal**

### CODE REVIEW

File: pkg/tsdbctl/tsdbctl.go [hunk 1]
```diff
@@ -122,11 +124,19 @@ func (rc *RootCommandeer) populateConfig(cfg *config.V3ioConfig) error {
 		}
 
 		slash := strings.LastIndex(rc.v3ioPath, "/")
-		if slash == -1 || len(rc.v3ioPath) <= slash+1 {
-			return fmt.Errorf("missing container name in V3IO URL")
+
+		if rc.container != "" {
+			cfg.Container = rc.container
+		} else {
+			if slash == -1 || len(rc.v3ioPath) <= slash+1 {
```
Comment (imakhlin) [inline pkg/tsdbctl/tsdbctl.go]: `len(rc.v3ioPath) <= slash+1` === `len(rc.v3ioPath) < slash`

Comment (gtopper) [reply pkg/tsdbctl/tsdbctl.go]: I thought `len(rc.v3ioPath) <= slash+1` ≈ `len(rc.v3ioPath) < slash+2`, @imakhlin.

File: pkg/tsdbctl/tsdbctl.go [hunk 2]
```diff
@@ -64,6 +65,7 @@ func NewRootCommandeer() *RootCommandeer {
 	cmd.PersistentFlags().StringVarP(&commandeer.dbPath, "dbpath", "p", "", "sub path for the TSDB, inside the container")
 	cmd.PersistentFlags().StringVarP(&commandeer.v3ioPath, "server", "s", defaultV3ioServer, "V3IO Service URL - username:password@ip:port/container")
 	cmd.PersistentFlags().StringVarP(&commandeer.cfgFilePath, "config", "c", "", "path to yaml config file")
+	cmd.PersistentFlags().StringVarP(&commandeer.container, "container", "u", "", "container to use")
```
[the remark this answers is not in the document]
Comment (dinal) [reply pkg/tsdbctl/tsdbctl.go]: no, if it's not passed here it will be derived from server url

File: pkg/tsdbctl/tsdbctl.go (hunk 1 above)
Comment (dinal) [reply pkg/tsdbctl/tsdbctl.go]: yep, what topper said

File: pkg/tsdbctl/tsdbctl.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (imakhlin) [reply pkg/tsdbctl/tsdbctl.go]: note, you can always override the default

**Pull request merged**